addons/survey/views/survey_templates_statistics.xml

```diff
@@ -99,46 +86,82 @@
-                            <span class="badge d-flex gap-1 ms-0 me-1 text-info text-start border rounded">
+                <div t-attf-class="d-flex flex-wrap align-items-center mb-1 collapse show o_survey_results_question_#{question.id}">
+                    <!-- Question info -->
+                    <div class="d-flex align-items-center my-1 me-2">
+                        <!-- Scoring info -->
+                        <t t-if="survey.scoring_type != 'no_scoring' and (question.answer_numerical_box or question.answer_date or
+                            question.answer_datetime or any([a.is_correct for a in question.suggested_answer_ids]))">
+                            <span t-if="question.question_type in ['numerical_box', 'date', 'datetime', 'simple_choice', 'multiple_choice']"
+                                class="badge d-flex gap-1 ms-0 me-1 text-info text-start border rounded">
+                                <span t-out="question_data['right_inputs_count']"/>
+                                <span class="text-success">Correct</span>
+                            </span>
+                            <span t-if="question.question_type in ['multiple_choice']" class="badge d-flex gap-1 ms-0 me-1 text-info text-start border rounded">
                                 <span t-esc="question_data['partial_inputs_count']"/>
                                 <span class="text-warning">Partial</span>
                             </span>
                         </t>
-                    </t>
-                    <!-- Inputs info -->
-                    <span class="badge d-flex gap-1 ms-0 me-1 text-info text-start border rounded">
-                        <span t-out="len(question_data['answer_input_done_ids'])"/>
-                        <span class="text-muted">Responded</span>
-                    </span>
-                    <span class="badge d-flex gap-1 ms-0 me-1 text-info text-start border rounded">
-                        <span t-out="len(question_data['answer_input_skipped_ids'])"/>
-                        <span class="text-muted">Skipped</span>
+                        <!-- Inputs info -->
+                        <span class="badge d-flex gap-1 ms-0 me-1 text-info text-start border rounded">
+                            <span t-out="len(question_data['answer_input_done_ids'])"/>/
+                            <span t-out="len(question_data['answer_line_ids'].mapped('user_input_id'))"/>
+                            <span class="text-muted">Responded</span>
+                        </span>
+                    </div>
+                    <!-- Numerical KPIs -->
+                    <span t-if="question.question_type in ['numerical_box', 'scale']" class="d-flex gap-1 my-1 me-2">
+                        <div class="btn-group o_survey_results_question_pill" role="group" aria-label="Maximum">
+                            <span class="badge text-bg-secondary only_left_radius px-2 pt-1">Max </span>
```

Comment:
While we're at it, I would remove the extra space we put after these labels and add a proper padding/margin instead, as it's "dangerous" to leave blank spaces in translated terms, translators might remove them.